logiclearner/main/views.py

```diff
@@ -19,3 +23,27 @@ class HintApiView(APIView):
     @method_decorator(csrf_exempt)
     def dispatch(self, *args, **kwargs):
         return super(HintApiView, self).dispatch(*args, **kwargs)
+
+
+class StatementListAPIView(generics.ListAPIView):
+    serializer_class = StatementSerializer
+
+    def get_queryset(self):
+        """
+        This view should return a list of all the statements for
+        the difficulty level determined by the difficulty portion of the URL.
+        """
+        difficulty = self.kwargs['difficulty']
```

Comment:
Consider adding a default here, a la `self.kwargs.get('difficulty', 'default value')`. This will prevent 500 errors when difficulty is not passed.

Reply:
@sdreher got it!